Public/Sdk/Public/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -126,16 +143,31 @@ export function runQTest(args: QTestArguments): Result {
         const qTestContextInfoFile = Environment.getFileValue("[Sdk.BuildXL]qtestContextInfo");
         qTestContextInfoPath = qTestContextInfoFile.path;
     }
-     
+    // TODO: Renaming the internal flag passing from GBR, will remove the old one when the new one roll out from GBR
+    else if (Environment.hasVariable("[Sdk.BuildXL.CBInternal]qtestContextInfo")){
+        const qTestContextInfoFile = Environment.getFileValue("[Sdk.BuildXL.CBInternal]qtestContextInfo");
+        qTestContextInfoPath = qTestContextInfoFile.path;  
+    }

```

Comment:
>} [](start = 4, length = 1)

This can be shorten to
```
const qTestContextInfoPath = Environment.getPathValue("[Sdk.BuildXL]qtestContextInfo") || Environment.getPathValue("[Sdk.BuildXL.CBInternal]qtestContextInfo")
```